package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mixer/retrieval",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
This is a breaking change, the version should be revved to 2.0.0

Reply:
done